app-header/app-header.html

```diff
@@ -217,7 +217,13 @@
           type: Boolean,
           reflectToAttribute: true,
           value: false
+        },
+
+        blockScrollEffects: {
```

Comment:
this property is doing a little more than blocking the scroll effects. Should we rename this property as `disabled`?  wdyt?